data-reconciliation-app/Makefile

```diff
@@ -58,4 +58,10 @@ e2e-test: ## 📦 Run end to end tests
 
 unit-test: ## 🔨 Run the Application unit-test
 	@echo -e "\e[34m$@\e[0m" || true
-	@npm run unit-test
\ No newline at end of file
+	@npm run unit-test
+	
+demo-code: ## Demo
```

Comment:
```suggestion
demo: ## Demo
```

Reply:
This I have kept intentionally as we have a demo folder in our app, hence make demo doesn't work in that case unless I do a build first.

Reply:
I think you should build before you demo anyway - so you could have 
```
demo: build
```

Reply:
Yes, we will be doing start-host which has build and then in a new terminal we can run make demo-code.

Reply:
Ah ok - I see what your intent is - I think we need to say somewhere (like in the banking sample)

# Run `make start-host` in another terminal before running this script
# To build and start a ccf network and automatically deploy your application on it.


- Also in the Banking-app sample - all of our scripts are kept in the root "scripts" folder

Reply:
Added.